src/main/java/com/aws/greengrass/util/platforms/unix/UnixExec.java

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.aws.greengrass.util.platforms.unix;
+
+import com.aws.greengrass.logging.api.Logger;
+import com.aws.greengrass.logging.impl.LogManager;
+import com.aws.greengrass.util.platforms.Exec;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import javax.annotation.Nullable;
+
+@SuppressWarnings("PMD.AvoidCatchingThrowable")
+public class UnixExec extends Exec {
+    private static final Logger logger = LogManager.getLogger(UnixExec.class);
```

Comment:
don't confuse things. This is a static logger, we also keep a logger which is specific to the instance of `Exec`

Reply:
Rename to staticLogger to keep that distinction very clear